### PR TITLE
specifies the temp directory used by pyinstaller …

### PR DESCRIPTION
**Pull request by shamsimam**
…to unpackage the cook executor

### CODE REVIEW

File: executor/Dockerfile.build [hunk 1]
```diff
@@ -10,4 +10,4 @@ RUN pip install -r requirements.txt
 
 COPY . /opt/cook
 
-CMD ["pyinstaller", "-F", "-n", "cook-executor", "-p", "cook", "cook/__main__.py"]
+CMD ["pyinstaller", "--runtime-tmpdir", ".cook-executor", "-F", "-n", "cook-executor", "-p", "cook", "cook/__main__.py"]
```
Comment (dposada) [inline executor/Dockerfile.build]: What is this relative to? Should we make it absolute?

Comment (dposada) [reply executor/Dockerfile.build]: (same question for the other places)

Comment (shamsimam) [reply executor/Dockerfile.build]: We want it to be relative to the sandbox directory, we don't have access to that path while running this command. I can add a comment in the Cook executor README if that helps.

**Comment by shamsimam** [comment]
@dposada build is green now

**Pull request merged**